### PR TITLE
Fixes #18875: Error when parsing change-validation.conf file

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/18875

### CODE REVIEW

File: change-validation/src/main/resources/change-validation.conf [hunk 1]
```diff
@@ -1,7 +1,6 @@
 # Email server configuration
 smtp.hostServer=""
-smtp.port=
-smtp.auth=true
```
Comment (ElaadF) [inline change-validation/src/main/resources/change-validation.conf:4]: This is never used

File: change-validation/src/main/resources/change-validation.conf [hunk 2]
```diff
@@ -1,7 +1,6 @@
...
+smtp.port=587
```
Comment (ElaadF) [inline change-validation/src/main/resources/change-validation.conf:3]: Have to put a default port to prevent error parsing

**Review by fanf — APPROVED** [review]

**Comment by ElaadF** [comment]
OK, merging this PR

**Comment by ElaadF** [comment]
OK, merging this PR

**Comment by ElaadF** [comment]
OK, merging this PR

**Comment by ElaadF** [comment]
OK, merging this PR

**Comment by ElaadF** [comment]
PR rebased

**Comment by ElaadF** [comment]
OK, merging this PR

**Comment by ElaadF** [comment]
OK, merging this PR

**Pull request merged**